census/customestimator/trainer/task.py

```diff
@@ -200,32 +171,6 @@ def _experiment_fn(run_config, hparams):
   os.environ['TF_CPP_MIN_LOG_LEVEL'] = str(
       tf.logging.__dict__[args.verbosity] / 10)
 
-  # If job_dir_reuse is False then remove the job_dir if it exists
-  if not args.reuse_job_dir:
-    if tf.gfile.Exists(args.job_dir):
-      tf.gfile.DeleteRecursively(args.job_dir)
-      tf.logging.info("Deleted job_dir {} to avoid re-use".format(args.job_dir))
-    else:
-      tf.logging.info("No job_dir available to delete")
-  else:
-    tf.logging.info("Reusing job_dir {} if it exists".format(args.job_dir))
-
-
   # Run the training job
-  # learn_runner pulls configuration information from environment
-  # variables using tf.learn.RunConfig and uses this configuration
-  # to conditionally execute Experiment, or param server code
-  learn_runner.run(
-      generate_experiment_fn(
-          min_eval_frequency=args.min_eval_frequency,
-          eval_delay_secs=args.eval_delay_secs,
-          train_steps=args.train_steps,
-          eval_steps=args.eval_steps,
-          export_strategies=[saved_model_export_utils.make_export_strategy(
-              model.SERVING_FUNCTIONS[args.export_format],
-              exports_to_keep=1
-          )]
-      ),
-      run_config=tf.contrib.learn.RunConfig(model_dir=args.job_dir),
-      hparams=hparam.HParams(**args.__dict__)
-  )
+  hparams=hparam.HParams(**args.__dict__)
+  run_experiment(hparams)
```

Comment:
Does anything in the "customestimator" need to change wrt the ServingInputRecievers (see other comments on `estimator` example)

Reply:
Yes it also needs to change. See ServingInputReceiver changes.